os/arch/arm/include/stm32h745/boardctl.h

```diff
@@ -0,0 +1,24 @@
+/****************************************************************************
```

Comment:
Minor - might wanna consider changing the header to say "2023" in case the merging takes time

Reply:
Thank you, If it is not merged 30 days ago, I 'll check this one more.

Reply:
I changed, Thanks